src/docs/markdown/build.md

```diff
@@ -26,8 +40,17 @@ Due to [a bug in Go](https://github.com/golang/go/issues/29228), these basic ste
 
 </aside>
 
+Go programs are easy to compile for other platforms. Just set the `GOOS`, `GOARCH`, and/or `GOARM` environment variables that are different. ([See the go documentation for details.](https://golang.org/doc/install/source#environment))
+
+For example, to compile Caddy for Windows when you're not on Windows:
+
+<pre><code class="cmd bash">GOOS=windows go build</code></pre>
+
+Or similarly for Linux ARMv6 when you're not on Linux or on ARMv6:
+
+<pre><code class="cmd bash">GOOS=linux GOARCH=arm GOARM=6 go build</code></pre>
 
-## xcaddy
+## Build using `xcaddy`
```

Comment:
I think changing this header will break a bunch of old `#xcaddy` anchor links in the forums etc.

Something we can do is add a tiny little `<script>` at the top of the page which just does like:
```html
<script>
if (window.location.hash == '#xcaddy') window.location.hash = '#build-using-xcaddy';
</script>
```

Would redirect the user.

Reply:
If the approach proposed in the [comment above](https://github.com/caddyserver/website/pull/370#discussion_r1474056196) is okay, there will be no breaking links. I can include this code snippet if the section ID ends up being differnt.